npm install oc 0.38.7 fails

Hi,

Just tried `npm i oc -g` and it tries to install v 0.38.7. It fails with the following error:

```
npm install oc -g
/Users/x/.nvm/versions/node/v6.9.5/bin/oc -> /Users/x/.nvm/versions/node/v6.9.5/lib/node_modules/oc/src/oc-cli.js

> oc@0.38.7 postinstall /Users/x/.nvm/versions/node/v6.9.5/lib/node_modules/oc
> grunt build

grunt-cli: The grunt command line interface (v1.2.0)

Fatal error: Unable to find local grunt.

If you're seeing this message, grunt hasn't been installed locally to
your project. For more information about installing and configuring grunt,
please see the Getting Started guide:

http://gruntjs.com/getting-started
- defined@1.0.0 node_modules/oc/node_modules/defined
- acorn@5.1.1 node_modules/oc/node_modules/falafel/node_modules/acorn
- async@1.5.2 node_modules/oc/node_modules/handlebars/node_modules/async
- isarray@1.0.0 node_modules/oc/node_modules/isobject/node_modules/isarray
- isarray@1.0.0 node_modules/oc/node_modules/node-libs-browser/node_modules/isarray
- buffer@4.9.1 node_modules/oc/node_modules/node-libs-browser/node_modules/buffer
- isarray@1.0.0 node_modules/oc/node_modules/readable-stream/node_modules/isarray
- acorn@5.1.1 node_modules/oc/node_modules/webpack/node_modules/acorn
- detective@4.5.0 node_modules/oc/node_modules/detective
npm ERR! Darwin 16.6.0
npm ERR! argv "/Users/x/.nvm/versions/node/v6.9.5/bin/node" "/Users/x/.nvm/versions/node/v6.9.5/bin/npm" "install" "oc" "-g"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! oc@0.38.7 postinstall: `grunt build`
npm ERR! Exit status 99
npm ERR!
npm ERR! Failed at the oc@0.38.7 postinstall script 'grunt build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the oc package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs oc
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls oc
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/x/Documents/projects/components/header/npm-debug.log
```

Can't understand why I need local grunt for!!! This seems to be a bug on this version

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -10,7 +10,7 @@ version: "{build}"
 # what combinations to test
 environment:
   matrix:
-    - nodejs_version: 4.7.1
+    - nodejs_version: 6.11.0
 
 # Get the stable version of node
 install:
@@ -24,4 +24,4 @@ install:
 build: off
 
 test_script:
-  - cmd: grunt test
+  - cmd: npm test

--- a/package.json
+++ b/package.json
@@ -7,9 +7,8 @@
     "oc": "./src/oc-cli.js"
   },
   "scripts": {
-    "postinstall": "grunt build",
     "precommit": "lint-staged",
-    "test": "grunt test"
+    "test": "grunt build && grunt test"
   },
   "lint-staged": {
     "*.js": [

--- a/src/components/oc-client/.npmignore
+++ b/src/components/oc-client/.npmignore
@@ -1,4 +1,2 @@
 package.tar.gz
 node_modules
-src
-_package

